drivers/timer/Kconfig.nrf_grtc

```diff
@@ -20,6 +20,11 @@ config NRF_GRTC_ALWAYS_ON
 	help
 	  Always keep the SYSCOUNTER active even if the CPU is in sleep mode.
 
+config NRF_GRTC_CLEAR_AT_INIT
+	bool "Clear the GRTC on driver init."
+	help
+	  Clear the GRTC counter on initialization.
```

Comment:
Since in https://github.com/zephyrproject-rtos/zephyr/issues/89147#issuecomment-2863336846 you say that clearing the GRTC is not something we want to do, this should probably mention the consequences of doing so.